features/custom_stack_trace.feature

```diff
@@ -1,5 +1,6 @@
 Feature: Custom stack trace
 
+  @spawn
```

Comment:
Without this as spawn, an issue where we weren't handling this correctly (calling toString on undefined line) led to this leaking to any following features